src/Toolkit/Str.php

```diff
@@ -315,6 +317,13 @@ public static function date(
 			return $format->format($time ?? time());
 		}
 
+		// automatically determine the handler from global configuration
+		// if an app instance is already running; otherwise fall back to
+	 	// `date` for backwards-compatibility
+		if ($handler === null) {
+			$handler = App::instance(null, true)?->option('date.handler') ?? 'date';
+		}
```

Comment:
Sorry for the multiple suggestions

```suggestion
		$handler ??= App::instance(null, true)?->option('date.handler') ?? 'date';
```

Reply:
It's indeed shorter and does the same, but here I'd say the current version is more readable as it visually fits quite well with the `if null`, `if intl`, `if strftime` blocks. But I'd be fine with the change.

Reply:
I suggested it because we have been using the `??=` syntax a lot lately. I'm OK with both. Feel free apply or discard 👍

Reply:
I'm sure @distantnative will agree with you. I'm fine with it either way, but I prefer the current version for the mentioned reason. :)

Reply:
I'm sure Nico will even update this line with `??=` when he refactor some parts 😄

Reply:
I agree that the readability is better like this. But I also really don't mind too much. Both ways work for me.